tests/pipelines/eks/upstream-load.yaml

```diff
@@ -0,0 +1,109 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: loadtest
+spec:
+  workspaces:
+    - name: config
+    - name: source
+    - name: results
+  params:
+  - name: cluster-name
+    description: The name of the EKS cluster you want to spin.
+  - name: eks-version
+    default: "1.20"
+    description: The EKS version to install.
+  - name: region
+    description: The region where the cluster is in.
+    default: "us-west-2"
+  - name: zones
+    description: The zones where the cluster is in.
+    default: "us-west-2a,us-west-2b,us-west-2c"
+  - name: nodegroup-name
+    default: "linux-nodes"
+    description: The name of the nodegroup of the cluster.
+  - name: instance-selector-vcpus
+    default: 2
+    description: The vcpus of the EC2 instaces for the nodegroup of the cluster.
+  - name: instance-selector-memory
+    default: 4
+    description: The memory of the EC2 instaces for the nodegroup of the cluster.
+  - name: desired-nodes
+    default: "100"
+    description: The desired number of nodes in the cluster.
+  - name: min-nodes
+    default: "1"
+    description: The minimum number of nodes in the cluster.
+  - name: max-nodes
+    default: "100"
+    description: The maximum number of nodes in the cluster.
+  - name: giturl
+    description: "git url to clone the package"
+    default: https://github.com/kubernetes/perf-tests.git
+  - name: pods-per-node
+    description: "pod density"
+    default: "10"
+  - name: nodes-per-namespace
+    description:  "nodes per namespace to get created for load test "
+    default: "10"
```

Comment:
Thoughts on bumping this to 100? ~1000 pods per namespace seems realistic. 100 namespaces in the cluster also seams realistic.

Reply:
default we have - 1k nodes, which means we will have 100 namespaces as per the test config if I'm doing my Math correctly, which is what you are also asking I guess :D